flaskr/app.py

```diff
@@ -56,4 +61,32 @@ class Method(db.Model):
     language_id = db.Column(db.Integer, db.ForeignKey('languages.id'))
 
     def __repr__(self):
-        return '<Method %r>' % self.name
\ No newline at end of file
+        return '<Method %r>' % self.name
+
+# schema objects
+class MethodObject(SQLAlchemyObjectType):
+    class Meta:
+        model = Method
+        interfaces = (graphene.relay.Node, )
```

Comment:
A note for future self, we included relays but we did not *really* need them... but most tutorials included them regardless.